subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalBuildIntegrationTest.groovy

```diff
@@ -1283,14 +1285,10 @@ task generate(type: TransformerTask) {
     }
 
     @ToBeImplemented("Private getters should be ignored")
```

Comment:
I think the `ToBeImplemented` can go away now, and the test name should be changed.

Reply:
I was under the impression that this `@TBI` test is doing what it should. This is also why I don't like `@ToBeImplemented` compared to the usual `@NotYetImplemented`: from my understanding this test expects that we _can_ override a private setter. If I remove the annotation, then it proves that we _can't_, which is exactly the opposite behavior.

With `@NotYetImplemented` you would actually write the expected test code, which is much clearer. So: what do we expect? /cc @lptr

Reply:
I've read the test again (the original one) and it's still unclear to me what was the intended behavior.